### DISCUSSION TITLE
Setting up tutorial repository

### DISCUSSION THREAD

**Discussion by Draper18** [idaholab/moose#17532] [Q&A Getting Started] [answered]
Hello, 
I am trying to get a repository set up so that I can continue with the moose tutorial but I am having some problems. It made the babbler folder but there is nothing in it. Below is the error messages that I am receiving. Any help is greatly appreciated!
![Screenshot from 2021-04-07 11-30-06](https://user-images.githubusercontent.com/81650298/113909940-7ee83300-9795-11eb-8a6f-2ee9ed37fe2b.png)

**Comment by cticenhour** [comment, marked as answer]
The error suggests that your projects directory has been made into a git repository somehow, as the script runs the command `git status` to check for that. If you run `ls -la ~/projects` in your Terminal, do you see a `.git` directory listed?

**Reply by Draper18** [reply]
Nothing that looks like a git repository. I do have the babbler folder set up as a git repository. Could that be the reason for the error? 


![Screenshot from 2021-04-07 12-08-52](https://user-images.githubusercontent.com/81650298/113913992-37b07100-979a-11eb-8e32-7f02a263ea98.png)

**Reply by cticenhour** [reply]
No, the stork script is meant to create a directory and initialize it as a git repository before filling in the things you need to set up a basic app. If `.git` doesn't exist in `projects` , then the `git status` command should throw `fatal: not a git repository (or any of the parent directories): .git` in that directory, which then prompts the stork script to proceed. You might perform a sanity check that this is the case by running `git status` in `~/projects`. 

Next thing to try - just to confirm something weird isn't going on in your Terminal instance, close the Terminal window you have open and start a new one. While in `~/projects` I would perform the command `rm -rf babbler` to remove the babbler directory altogether and try running the stork script again. If this gives you the same error, we'll need to dive a little deeper to figure out what's going on.

**Reply by cticenhour** [reply]
One more thing that just occurred to me. This error could also occur if your home directory was initialized as a git repo (since projects exists within that directory). Run `ls -la ~` in your Terminal - do you see a `.git`?

**Reply by Draper18** [reply]
When I run the ls -la ~ I do see a .git


![Screenshot from 2021-04-07 14-57-51](https://user-images.githubusercontent.com/81650298/113933374-bc5ab980-97b1-11eb-94ce-8d9a56088382.png)

**Reply by cticenhour** [reply]
This is the root cause of the error. Your home directory should not be made into a git repository, so I would recommend removing the `~/.git` directory:
```
rm -rf ~/.git
```
After removing it, you should be able to try using the stork script again.

**Reply by cticenhour** [reply]
@Draper18 just checking in - were you able to get things going with that suggestion?
